src/services/posts/assign-tag-to-post.test.js

```diff
@@ -0,0 +1,117 @@
+const {expect} = require('chai')
```

Comment:
Its either linting has been disabled or we do not have rules.
Can you kindly fix the lining over here? eg `{ expect }`
Let's do same for the others